data-prep/app/routes.py

```diff
@@ -1,9 +1,21 @@
 from app import app,APP_ROOT
 
+from app.model import hello
+
 @app.route("/")
 def home():
     return "hello world"
 
 @app.route("/predict")
 def predict():
     return "SUPER CLEVER AI DATA"
+
+@app.route("/featureAnalysis")
+def featureAnalysis():
+    ms = hello
+    return ms
```

Comment:
So these functions need to return a string, so we possible have the data encoded into JSON.